tools/istio-iptables/pkg/dependencies/implementation_linux.go

```diff
@@ -261,22 +261,22 @@ func (r *RealDependencies) executeXTablesWithOutput(log *log.Scope, cmd constant
 			c = exec.Command(cmdBin, args...)
 		}
 	}
-	log.Infof("Running command (%s): %s %s", mode, cmdBin, strings.Join(args, " "))
+	if quietLogging {
```

Comment:
IMO just remove this check, and make the informational echo always `Debug`.

Then the flag is just `ignoreErrors` or not.

Reply:
I think it can be moved to debug since we already print
log.Infof("Running iptables restore with: %s and the following input: ...)
in the run.go.
Let me try and see how the logs look afterwards